src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -169,10 +169,13 @@ public void keyboardShortcutsTest() {
         // test shortcut on repo dropdown
         doubleClick(repoSelectorComboBox);
         pushKeys(SWITCH_DEFAULT_REPO);
-        waitUntilNodeAppears("#dummy/dummy_col1_1");
+        // wait for issue 9 to appear then click on it
+        // issue 9 is chosen instead of issue 10
+        // as there is a problem with finding issue 10's node due to it being the first card in the panel
+        waitUntilNodeAppears("#dummy/dummy_col1_9");
```

Comment:
Highly certain this is not a case of mishandling double-digit numbers.
In the context of #1260, where the number of dummy issues have been increased from 10 to 12, it still fails when trying to find issue 12, while still being successful when the issue to check for is changed from 12 to 11.

The following debug statements also suggest that this is not due to the indexing problems. The node with the id `dummy/dummy_col1_12` is produced, but cannot be found.
![screen shot 2016-02-28 at 3 34 02 pm](https://cloud.githubusercontent.com/assets/12193772/13378051/15f3eb50-de31-11e5-8083-e4170a37b733.png)